hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/ConfigurationKeys.java

```diff
@@ -130,5 +130,12 @@ public static String accountProperty(String property, String account) {
 
   public static final String ABFS_EXTERNAL_AUTHORIZATION_CLASS = "abfs.external.authorization.class";
 
+  /** Key to enable custom identity transformation */
```

Comment:
nit: add a "." to the end of each sentence, here and below, to keep javadoc happy